rotkehlchen/tests/unit/test_price_historian.py

```diff
@@ -0,0 +1,233 @@
+from enum import Enum
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from rotkehlchen.constants.assets import A_BTC, A_GBP, A_USD
+from rotkehlchen.constants.misc import ZERO
+from rotkehlchen.errors import NoPriceForGivenTimestamp, PriceQueryUnsupportedAsset, RemoteError
+from rotkehlchen.externalapis.coingecko import Coingecko
+from rotkehlchen.externalapis.cryptocompare import Cryptocompare
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.price import PriceHistorian
+from rotkehlchen.history.typing import (
+    DEFAULT_HISTORICAL_PRICE_ORACLE_ORDER,
+    HistoricalPriceOracle,
+    HistoricalPriceOracleProperties,
+)
+from rotkehlchen.typing import Price, Timestamp
+
+
+@pytest.fixture(name='oracle_order')
+def fixture_oracle_order():
+    return DEFAULT_HISTORICAL_PRICE_ORACLE_ORDER
+
+
+@pytest.fixture(name='price_historian')
+def fixture_price_historian(oracle_order):
```

Comment:
This is only for this file, right? You are not exporting it anywhere else? Cause we already got a price historian fixture in `tests/fixtures/history.py`

Reply:
Yes, only for this test. I've renamed it as `fake_price_historian`.